Dockerfile

```diff
@@ -65,9 +63,6 @@ WORKDIR /
 # Install operator binary to WORKDIR
 COPY --from=builder ${DEST_ROOT}/manager .
 
-# Install templates
-COPY --from=builder ${DEST_ROOT}/templates ${OPERATOR_TEMPLATES}
-
```

Comment:
Thanks for removing we don't need it and it causes the error below (unless you mkdir to trick it).  
```
Error: error building at STEP "COPY --from=builder ${DEST_ROOT}/templates ${OPERATOR_TEMPLATES}": checking on sources under "/home/fultonj/.local/share/containers/storage/overlay/8fef8694792ebe871262ce7148a9c9e02e6ef45aea5465f0640d454c7727df7e/merged": copier: stat: "/dest-root/templates": no such file or directory
```